pkg/kyverno/common/common.go

```diff
@@ -737,25 +737,28 @@ func GetResourceAccordingToResourcePath(fs billy.Filesystem, resourcePaths []str
 			}
 		} else if (len(resourcePaths) > 0 && resourcePaths[0] != "-") || len(resourcePaths) < 0 || cluster {
```

Comment:
Is there a case when none of these conditions [ (len(resourcePaths) > 0 && resourcePaths[0] != "-") || len(resourcePaths) < 0 || cluster ] are true? What happens then?

If that is not possible, we can remove the conditions and change the `else if` to an `else`.

Reply:
correct, 
we not required [ (len(resourcePaths) > 0 && resourcePaths[0] != "-") || len(resourcePaths) < 0 || cluster ] condition like this.
I have minimised the condition. working as expected.